weave/Scripts/Multiplayer/Manager.cs

```diff
@@ -0,0 +1,204 @@
+using Godot;
+using System;
+using SIPSorcery.Net;
+using Newtonsoft.Json;
+using Weave.InputSources;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Net.WebSockets;
+using System.Text;
+
+namespace Weave.Multiplayer;
```

Comment:
Im thinking if we should rename this folder/namespace to something like Networking, as it is multiplayer but the other input sources are also forms of multiplayer